lib/msal-node/src/cache/NodeStorage.ts

```diff
@@ -29,11 +29,19 @@ export class NodeStorage extends CacheManager {
     // Cache configuration, either set by user or default values.
     private logger: Logger;
     private cache: CacheKVStore = {};
+    private inMemoryCache: InMemoryCache;
```

Comment:
I’m not sure if this makes sense, since right above this, you have a private `cache` 🤔 

I could be wrong, but you already have a `setCache` and `getCache` methods in this file, meaning that there’s already a way to keep track of an “in memory cache” through this class. Is the idea to keep track of _two_ in memory caches?

I believe the utility of the method `cacheToInMemoryCache` was to convert from one type to another, not to commit cache to memory.